src/routes/docs/tutorials/nuxt/step-4/+page.markdoc

```diff
@@ -0,0 +1,175 @@
+    navigateTo("/");
+  };
+
+  const logout = async () => {
+    await account.deleteSession("current"); // Delete Appwrite user session
+    current.value = null; // Clear current ref
+    isLoggedIn.value = false; // Set ref to false
+    navigateTo("/");
+  };
+
+  return {
+    current,
+    isLoggedIn,
+    login,
+    logout,
+    register,
+  };
+};
+
+```
+# Login page {% #login-page %}
+
+To get the user input we will build a simple form with the logic from the `useUserSession()` composable.
+The form will have two input fields, one for the user's email-address and one for the password.
+Underneath will be two buttons for logging in or registering the user.
+
+Create a new file in the `pages` directory called `login.vue`.
+This will not only create a new page, it will also add the route `/login` to the url.
+In step 5 we will add a loginbutton that will redirect us to this page.
+Add the following code to build the form.
+
+```vue
+// pages/login.vue
+
+<template>
+  <div class="u-max-width-650" style="margin: 0 auto;">
+    <section class="card u-margin-32">
+      <h2 class="eyebrow-heading-2">Login/Register</h2>
+      <form
+        class="form u-width-full-line u-max-width-500 u-margin-block-start-16"
+        @submit.prevent="handleLogin || handleRegistration"
+      >
+        <ul class="form-list">
+          <!-- Input field e-mail -->
+          <li class="form-item">
+            <label class="label">Email</label>
+            <div class="input-text-wrapper">
+              <input
+                v-model="userData.email"
+                type="email"
+                class="input-text"
+                placeholder="Email"
+                required
+              />
+            </div>
+          </li>
+          <!-- Input field e-mail -->
+          <li class="form-item">
+            <label class="label">Password</label>
+            <div class="input-text-wrapper">
+              <input
+                v-model="userData.password"
+                type="password"
+                class="input-text"
+                placeholder="Password"
+                required
+              />
+            </div>
+          </li>
+        </ul>
+        <ul class="buttons-list u-margin-block-start-16">
+          <!-- Login button  -->
+          <li class="buttons-list-item">
+            <button
+              class="button is-small u-margin-inline-start-4"
+              aria-label="Login"
+              @click="handleLogin"
+            >
+              Login
+            </button>
+          </li>
+          <!-- Register button  -->
+          <li class="buttons-list-item">
+            <button
+              class="button is-small is-secondary u-margin-inline-start-4"
+              aria-label="Register account"
+              @click="handleRegistration"
+            >
+              Register
+            </button>
+          </li>
+        </ul>
+      </form>
+    </section>
+  </div>
+</template>
+
+<script>
```

Comment:
This could be using `<script setup>`, which is the best way to do it (https://vuejs.org/api/sfc-script-setup.html)

Reply:
Done